packages/valory/skills/termination_abci/rounds.py

```diff
@@ -200,3 +200,9 @@ class PostTerminationTxAbciApp(AbciApp[Event]):
 TerminationAbciApp.transition_function[BackgroundRound] = {
     Event.TERMINATE: TransactionSubmissionAbciApp.initial_round_cls,
 }
+TerminationAbciApp.db_pre_conditions = {
```

Comment:
Actually, I need to move this app to the post termination app and not have it on the chained app